Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV TILEGENERATION_CONFIGFILE=/etc/tilegeneration/config.yaml \
 
 EXPOSE 80
 
-WORKDIR /project
+WORKDIR /etc/tilegeneration/
```

Comment:
We will have to fix all the projects using that image...

Reply:
Why ?
This didn't have any incidence ...